packages/react-aria-components/test/Calendar.test.js

```diff
@@ -328,7 +328,7 @@ describe('Calendar', () => {
     let grid = getByRole('application');
     expect(grid).toHaveAttribute('class', 'grid');
 
-    let cell = within(grid).getAllByRole('button')[7];
+    let cell = within(grid).getAllByRole('button')[8];
```

Comment:
Will this break again next month?

Reply:
Possibly, in my description, I recommended we eventually just set a specific date on the calendar so it can't drift with real time

Reply:
i was in the midst of yarn 4 changes yesterday when I ran into it, so went for fastest fix. I've pushed something a little better now